.github/workflows/smoke-test-release.yml

```diff
@@ -1,14 +1,14 @@
 name: Smoke test release
 on:
     release:
-        types: [published]
+        types: [released, prereleased]
```

Comment:
I noticed in the `7.4.1` and `7.5.0-beta.2` releases that this workflow wasn't triggered. I figured it's because these releases may have already been published, and were just updated to be a `pre-release` or a `release` later on. Therefore, the `published` release type wasn't triggered because they were already in a published state to begin with.

Setting the release type to `released` and `prereleased` I think would be more appropriate.

Reply:
> I figured it's because these releases may have already been published, and were just updated to be a `pre-release` or a `release` later on.

I'm not sure that this is the case, as I always make sure to select one of those two before publishing a release... but if `released` and `prereleased` trigger this workflow as expected, I think this should be okay.